src/Documentation/doc/advanced/corporation/optimal-selling-price-market-ta2.md

```diff
@@ -0,0 +1,108 @@
+  - Material: $ItemMultiplier = MaterialQuality + 0.001$
+  - Product: $ItemMultiplier = 0.5\ast(ProductEffectiveRating)^{0.65}$
+- Business factor:
+
+$$BusinessProduction = 1 + office.employeeProductionByJob["Business"]$$
+
+$${BusinessFactor = (BusinessProduction)}^{0.26} + \left( \frac{BusinessProduction}{1000} \right)$$
+
+- Advert factor:
+  - $AwarenessFactor = (Awareness + 1)^{IndustryAdvertisingFactor}$
+  - $PopularityFactor = (Popularity + 1)^{IndustryAdvertisingFactor}$
+
+$$RatioFactor = \begin{cases}Max(0.01,\frac{Popularity + 0.001}{Awareness}), & Awareness \neq 0 \newline 0.01, & Awareness = 0 \end{cases}$$
+
+$$AdvertFactor = (AwarenessFactor\ast PopularityFactor\ast RatioFactor)^{0.85}$$
+
+- Market factor:
+
+$$MarketFactor = Max\left( 0.1,\frac{Demand\ast(100 - Competition)}{100} \right)$$
+
+- Corporation's upgrade bonus: `SalesBots` bonus.
+- Division's research bonus: this is always 1. Currently there is not any research that increases the sales bonus.
+- `MarkupMultiplier`: initialize with 1.
+
+  - `SellingPrice` is the selling price that you set.
+  - With materials, if we set `SellingPrice` to 0, `MarkupMultiplier` is $10^{12}$ (check the formula below). Extremely high `MarkupMultiplier` means that we can sell all units, regardless of other factors. This is the fastest way to discard materials.
+  - If `(SellingPrice > MarketPrice + MarkupLimit)`:
+
+  $$MarkupMultiplier = \left( \frac{MarkupLimit}{SellingPrice - MarketPrice} \right)^{2}$$
+
+  - If item is material and `SellingPrice` is less than `MarketPrice`:
+
+$$MarkupMultiplier = \begin{cases}\frac{MarketPrice}{SellingPrice}, & SellingPrice > 0 \land SellingPrice < MarketPrice \newline 10^{12}, & SellingPrice \leq 0 \end{cases} $$
+
+## Optimal selling price
+
+As we can see with previous part, `MarkupMultiplier` is basically a penalty modifier if we set `SellingPrice` greater than `(MarketPrice + MarkupLimit)`, and we'll always do this. This means we need to find out highest possible `SellingPrice` while `MaxSalesVolume` is still equals to `ExpectedSalesVolume`.
+
+This is the reason why we should not bother with Market-TA1. It simply sets `SellingPrice = MarketPrice + MarkupLimit`. This means Market-TA1 sets a "safe" `SellingPrice` for us, it guarantees that we won't be penalized due to too high price. However, this "safe" `SellingPrice` is too low, and we can find a much higher `SellingPrice`.
+
+Formula:
+
+- Define:
+
+$$M = \ ItemMultiplier\ast BusinessFactor\ast AdvertFactor\ast MarketFactor\ast SaleBotsBonus\ast ResearchBonus$$
+
+- We want `MaxSalesVolume` equals `ExpectedSalesVolume`:
+
+$$MaxSalesVolume = ExpectedSalesVolume$$
+
+≡
+
+$$M\ast\left( \frac{MarkupLimit}{SellingPrice - MarketPrice} \right)^{2} = ExpectedSalesVolume$$
+
+≡
+
+$$\frac{MarkupLimit}{SellingPrice - MarketPrice} = \sqrt{\left( \frac{ExpectedSalesVolume}{M} \right)}$$
+
+≡
+
+$$SellingPrice = \frac{MarkupLimit}{\sqrt{\frac{ExpectedSalesVolume}{M}}} + MarketPrice$$
```

Comment:
If you invert the ratio inside the root here, you avoid the need for a nested fraction.